keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -864,16 +911,22 @@ These goals will help you determine what you need to measure (SLIs) in the next
 question.
 -->
 
+- `supplementalGroupsPolicy=Strict`: 100% of pods were scheduled into a node with the feature supported.
```

Comment:
I don't see any scheduler integration incorporated into the KEP. How will this happen in clusters where some nodes support this and some do not?

Reply:
Yeah, I described this topic in later section. Let me update this line like this.

```suggestion
- `supplementalGroupsPolicy=Strict`: 100% of pods were scheduled into a node with the feature supported. This KEP does NOT support scheduler integration. Please see the section "Are there any missing metrics that would be useful to have to improve observability of this feature?". 
```

Reply:
I think that's helpful, but perhaps you can also note up in the discussion of the feature and how it works, that users should target using a node label.